conformance/lang/expressions/regexp/regexp_semantics.balt

```diff
@@ -0,0 +1,719 @@
+Test-Case: output
+Description: Test basic regular expression.
+Labels: RegExp, string
```

Comment:
Shall we add `tagged-data-template-expr` label since regular expressions belong to this.

Reply:
Should also add `string:regExp`, `regexp:RegExp` labels. Check other places as well